Add execute_timeout to GQL Client in Python SDK (#6919)
fishtown-analytics/docs.getdbt.com: website/docs/docs/dbt-versions/release-notes.md

## February 2025

> - **Enhancement**: Improved stability in the [Python SDK](/docs/dbt-cloud-apis/sl-python) by increasing the default `execute_timeout` in GraphQL from 10s to 30s to reduce test flakiness (inconsistent results), particularly for the "execute query sync" endpoint. 

Author: ```suggestion
- **Enhancement**: The [Python SDK](/docs/dbt-cloud-apis/sl-python) has improved stability by increasing the default `execute_timeout` in GraphQL from 10s to 30s to reduce test flakiness (inconsistent results). This is particularly useful for the "execute query sync" endpoint. 
```

> - **Enhancement**: The [Python SDK](/docs/dbt-cloud-apis/sl-python) has improved stability by increasing the default `execute_timeout` in GraphQL from 10s to 30s to reduce test flakiness (inconsistent results). This is particularly useful for the "execute query sync" endpoint. 

Author: @serramatutu is the timeout configurable or is it set to 30 secs?

> - **Enhancement**: The [Python SDK](/docs/dbt-cloud-apis/sl-python) has improved stability by increasing the default `execute_timeout` in GraphQL from 10s to 30s to reduce test flakiness (inconsistent results). This is particularly useful for the "execute query sync" endpoint. 

Author: ```suggestion
- **Enhancement**: The [Python SDK](/docs/dbt-cloud-apis/sl-python) has improved stability by increasing the default `execute_timeout` in GraphQL from 10 seconds to 30 seconds to reduce test flakiness (inconsistent results). This is particularly useful for the "execute query sync" endpoint. 
```

> - **Enhancement**: The [Python SDK](/docs/dbt-cloud-apis/sl-python) has improved stability by increasing the default `execute_timeout` in GraphQL from 10 seconds to 30 seconds to reduce test flakiness (inconsistent results). This is particularly useful for the "execute query sync" endpoint. 

Author: ```suggestion
- **Enhancement**: The [Python SDK](/docs/dbt-cloud-apis/sl-python) has improved stability by increasing the default `execute_timeout` in GraphQL from 10 seconds to 30 seconds to reduce test flakiness (inconsistent test results). This is particularly useful for the "execute query sync" endpoint. 
```

> - **Enhancement**: The [Python SDK](/docs/dbt-cloud-apis/sl-python) has improved stability by increasing the default `execute_timeout` in GraphQL from 10 seconds to 30 seconds to reduce test flakiness (inconsistent test results). This is particularly useful for the "execute query sync" endpoint. 

Author: ```suggestion
- **Enhancement**: The [Python SDK](/docs/dbt-cloud-apis/sl-python) has improved stability by increasing the default `execute_timeout` in GraphQL from 10 seconds to 30 seconds to reduce test flakiness (inconsistent test results). This is useful for endpoints like the "execute query sync". 
```

> - **Enhancement**:  The [Python SDK](/docs/dbt-cloud-apis/sl-python) added a new timeout parameter to Semantic Layer client and to the underlying GraphQL clients to specify timeouts. Set timeout number or `total_timeout` in the global `TimeoutOptions` to control connect, execute and close timeouts granularly. `ExponentialBackoff.timeout_ms` is now deprecated.

Author: ```suggestion
- **Enhancement**:  The [Python SDK](/docs/dbt-cloud-apis/sl-python) added a new timeout parameter to Semantic Layer client and to underlying GraphQL clients to specify timeouts. Set timeout number or `total_timeout` in the global `TimeoutOptions` to control connect, execute and close timeouts granularly. `ExponentialBackoff.timeout_ms` is now deprecated.
```
